src/components/Card.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+type cardType = {
```

Comment:
prefer PascalCase buat type/interface

```suggestion
type CardType = {
```